### PR TITLE
6.11 Release notes

### PR DESCRIPTION
**Pull request by lzdanski**
Also includes
- https://github.com/astronomer/docs/pull/3839
- https://github.com/astronomer/docs/pull/3848

### CODE REVIEW

**Review by conorbev — APPROVED** [review]
lgtm

File: astro/release-notes.md [hunk 1]
```diff
@@ -25,6 +25,24 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+### Additional improvements
+
+- When using the GitHub Integration, you can now retry a failed deploy or trigger a Git deploy directly from your Deployment settings page in the Astro UI. Refer to the [GitHub Integration](https://www.astronomer.io/docs/astro/deploy-github-integration) for more information. <!--https://github.com/astronomer/docs/pull/3839-->
+- Now, at the end of your Astro Trial, your Deployments automatically hibernate. You can then add a credit card number to your user account to access your Deployments and Workspaces to wake your hibernating Deployment. After 30 days, your hibernating Deployments and data are deleted. See [Start a trial](trial.md) for more information about Astro Trial policies and available resources.<!--https://github.com/astronomer/astro/pull/21514, https://github.com/astronomer/astro/pull/21562-->
+- The ability to create, update, and delete Deployment API tokens is restricted to users with the Workspace Owner role. Read [Workspace user permissions](https://www.astronomer.io/docs/astro/user-permissions#workspace-roles) for more information. <!--https://github.com/astronomer/astro/pull/21736-->
```
[the remark this answers is not in the document]
Comment (paolaperaza) [reply astro/release-notes.md:40]: This is really kind of a bug fix but it's fine

File: astro/trial.md [hunk 2]
```diff
@@ -35,11 +35,23 @@ You're now ready to start deploying and running DAGs on Astro. See [Run your fir
...
+
+Your Astro trial provides you access to specific kinds of Astronomer Deployment resources.
+
+- One workspace per trial
+- Two Deployments per trial
+- A maximum of 5 workers per worker queue
+- No high-availability enabled
+- A5, A10, or A20 workers
+
+To access additional resources or functionality, you need to add a credit card number or other payment method. You can read more about Deployment resources available for regular Astro plans in [Configure Deployment resources](deployment-resources.md).
```
Comment (paolaperaza) [inline astro/trial.md:48]:
```suggestion
An Astro trial provides you a level of access to Deployment resources that Astro considers sufficient to run an initial number of Apache Airflow DAGs and ensure that Astro meets the needs of your Organization.

During a 14-day Astro trial, you can:

- Create one Workspace
- Create up to two Deployments
- Configure **Worker Count** to a maximum of 5 workers per worker queue
- Use A5, A10, or A20 workers

You cannot use high-availability (HA) mode during an Astro trial. To access additional resources or functionality, add a credit card number or other payment method to your Organization.

To learn more about Deployment resources and features available to Astro customers, see [Configure Deployment resources](deployment-resources.md). To learn more about Astro pricing, see [Pricing](https://www.astronomer.io/pricing/).
```

Comment (paolaperaza) [reply astro/trial.md:48]: Framing this positively instead of negatively

File: astro/trial.md [hunk 3]
```diff
@@ -35,11 +35,23 @@ You're now ready to start deploying and running DAGs on Astro. See [Run your fir
...
+
 ## After your trial
 
-After your 14-day trial ends, you can no longer access your Deployments and Workspaces from the Astro UI. You can still access your user account page and Astronomer support forms.
+After your 14-day trial ends, you can no longer access your Deployments and Workspaces from the Astro UI. Your Deployments are hibernating. Even with hibernating Deployments, you can still access your user account page and Astronomer support forms. If you provie a credit card number or extend your trial, you can access your Deployments and Workspaces again, where you can wake your Deployments from hibernation.
```
Comment (paolaperaza) [inline astro/trial.md:52]:
```suggestion
After your 14-day trial ends, you can no longer access your Workspace from the Astro UI and your Deployments enter [hibernation](deployment-resources.md#hibernate-a-development-deployment) for 30 days. You can still access your user account page and Astronomer support forms. To regain access to your Deployments and Workspace, you must enter a payment method or contact Astronomer to extend your trial. Once you enter a payment method, you can wake your Deployments from hibernation and continue to run Apache Airflow DAGs.
```

File: astro/trial.md [hunk 4]
```diff
@@ -35,11 +35,23 @@ You're now ready to start deploying and running DAGs on Astro. See [Run your fir
...
 
-After your trial ends, all Deployments are automatically deleted. Any code that you deployed to Astro will be lost. If you need additional time to evaluate Astro, or you need to copy your configuration for future use, you can:
+You have 30 days after your trial ends to extend it before all Deployments are automatically deleted. When your Deployments are deleted, any code that you deployed to Astro will be lost. If you need additional time to evaluate Astro, or you need to copy your configuration for future use, you can:
```
Comment (paolaperaza) [inline astro/trial.md:54]:
```suggestion
All Deployment configurations are preserved during hibernation for 30 days from the last day of your Astro trial. After 30 days, your Deployment and all of its metadata are permanently deleted. When your Deployments are deleted, any code that you deployed to Astro will be lost.

If you need additional time to evaluate Astro, or you need to copy your configuration for future use, you can:
```

**Review by paolaperaza — APPROVED** [review]
Coupla comments here, thanks @lzdanski

File: astro/release-notes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lzdanski) [reply astro/release-notes.md:40]: There was a request to communicate it this way.

File: astro/release-notes.md [hunk 5]
```diff
@@ -25,6 +25,24 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+<EnterpriseBadge/>
+
+Organization owners can now limit the IP addresses of users to control who can access the Astro UI or programmatically work with the Astro API for their organization. This allows you to restrict access to your Astro Organization to users on a VPN or to specific network ranges. See [Set up IP Access list](https://astronomer.io/docs/astro/ip-access-list).
+
+### Additional improvements
+
+- When using the GitHub integration, you can now retry a failed deploy or trigger a Git deploy directly from your Deployment settings page in the Astro UI. See [GitHub Integration](https://www.astronomer.io/docs/astro/deploy-github-integration) for more information.
+- Deployments at the end of an Astro trial now automatically hibernate and are not immediately deleted. If your trial has ended, add a credit card number to your Organization to access your Workspaces and wake your hibernating Deployment. While in hibernation, Deployment settings and configurations are preserved. If you do not enter a payment method within 30 days, your hibernating Deployments and all corresponding data are deleted. See [Start a trial](trial.md) for more information.
```
Comment (lzdanski) [inline astro/release-notes.md:39]:
```suggestion
- Deployments at the end of an Astro trial now automatically hibernate and are not immediately deleted. If your trial has ended, add a credit card number to your Organization to access your Workspaces and wake your hibernating Deployment. While in hibernation, Deployment settings and configurations are preserved. If you do not enter a payment method within 30 days, your hibernating Deployments and all corresponding data are deleted. See [Start a trial](https://www.astronomer.io/docs/astro/trial) for more information.
```

**Pull request merged**